lib/iceberg/events/life_stop.js

```diff
@@ -10,16 +10,14 @@
  */
 const onLifeStop = async (instance = {}) => {
   const { state = {}, h = {} } = instance
-  const { timeout } = state
-  const { debouncedSubmitOrders, debug, updateState } = h
+  const { orders = {}, gid } = state
+  const { debouncedSubmitOrders, emit, debug } = h
 
   debouncedSubmitOrders.cancel()
 
-  if (timeout !== null) {
-    clearTimeout(timeout)
-    await updateState(instance, { timeout: null })
-    debug('cleared timeout')
```

Comment:
you still have to clear the timeout from submit delays?

Reply:
thanks!! i will add this